extension/src/popup/components/account/AccountAssets.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
```

Comment:
this is sort of placeholder. This component will be completely changed when we add an asset + payment history UI next